src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -350,13 +350,12 @@ public async Task SslStream_UntrustedCaWithCustomCallback_Throws(bool customCall
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/46837", TestPlatforms.OSX)]
         public async Task SslStream_ClientCertificate_SendsChain()
         {
             List<SslStream> streams = new List<SslStream>();
             TestHelper.CleanupCertificates("SslStream_ClinetCertificate_SendsChain");
             (X509Certificate2 clientCertificate, X509Certificate2Collection clientChain) = TestHelper.GenerateCertificates("SslStream_ClinetCertificate_SendsChain", serverCertificate: false);
-            using (X509Store store = new X509Store(StoreName.CertificateAuthority, StoreLocation.CurrentUser))
+            using (X509Store store = new X509Store(PlatformDetection.IsOSXLike ? StoreName.My : StoreName.CertificateAuthority, StoreLocation.CurrentUser))
```

Comment:
Can we add a comment explaining why OSX is different?

Reply:
ok. I will. It is primarily #48207 but there still seems to be another trickery in CI. I'm looking into it and I'll hold this back until clean CI pass.